src/components/formik/FormikComponent.tsx

```diff
@@ -1,12 +1,12 @@
-import React from "react";
-import * as Yup from "yup";
-import { AnyObject } from "yup/lib/types";
-import { useHistory } from "react-router-dom";
-import Style from "./formikStyle.module.css";
-import FieldComponent from "./fieldComponents/FieldComponent";
-import { Formik, Form } from "formik";
-import { FormSection } from "./types";
-
+/* eslint-disable no-console */
```

Comment:
Could we change this to disable no-console for the next line only, since there is only one console log on line 27? I feel like it's easy to forget about it after disabling it for the entire file. The other components that had disabling lint for the entire file I was ok with since those didn't look like 'finished' components to me, but the `FormikComponent` is supposed to be done except for submitting

Reply:
I would like to leave this for ow.  Can you add an issue if you want?